### ISSUE TITLE
Allow to apply an opaque when drawing widget

### ISSUE THREAD

**Issue by Steve-xmh** [linebender/druid#1638]
It will be useful when making animation like fade in/out.

**Comment by cmyr** [comment]
I'm not sure I understand this question. An opaque background color?

**Comment by Steve-xmh** [comment]
> I'm not sure I understand this question. An opaque background color?

Yes, but apply on widget(Like #1631 I guess?), which means we won't need to use a rectangle with same background color to cover the widget we need to make a fade-in or fade-out animation on.

I'm a little poor in English, reply me if you still confusing. :D
